README.md

```diff
@@ -1,23 +1,39 @@
 # CS Club Website
 
-## Development
+---
 
-1. Install dependencies
+This is the official repository for the University of Adelaide Computer Science Club (CS Club) website. It is built using [Next.js](https://nextjs.org/), [React](https://reactjs.org/), [TypeScript](https://www.typescriptlang.org/), and [Tailwind CSS](https://tailwindcss.com/).
```

Comment:
I don't know if it's necessary to list out the tech stack we're using

Reply:
It's more so for new contributors who are interested in contributing to the project. Although, it might be better to put this info in `CONTRIBUTING.md`.